services/device-registry-legacy/src/main/java/org/eclipse/hono/deviceregistry/ApplicationConfig.java

```diff
@@ -215,6 +216,17 @@ public ServiceConfigProperties restProperties() {
         return props;
     }
 
+    /**
+     * Creates an authentication handler used by device registry management HTTP API.
+     *
+     * @return The handler.
+     */
+    @Bean
+    @Scope("prototype")
+    public AuthHandler authHandler() {
```

Comment:
Is this just a placeholder which you will fill in before merging this PR?
Otherwise I do not see the value of this, given that the auth handler is not a required property ...

Reply:
It was used for manual testing and I thought it'd be good to leave it as a placeholder for the future. I removed it as suggested.